README.rst

```diff
@@ -7,18 +7,21 @@
    This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
    either express or implied. See the License for the specific language governing permissions and
    limitations under the License.
+
 Welcome
 =======
 This is the repository for code examples used in the public 
-`AWS documentation <https://docs.aws.amazon.com>`_. The examples use the AWS SDKs for the supported programming languages. For more information, see `Tools to Build on 
-AWS <https://aws.amazon.com/getting-started/tools-sdks/>`_.
+`AWS documentation <https://docs.aws.amazon.com>`_. The examples use the AWS SDKs for the supported
+programming languages. For more information, see
+`Tools to Build on AWS <https://aws.amazon.com/getting-started/tools-sdks/>`_.
 
 Finding code examples
 =====================
 
-The code examples are organized by programming language. For example, all of the code examples for the V1 version of the
-`AWS SDK for Java Developer Guide <https://www.github.com/awsdocs/aws-java-developer-guide>`_ are
-kept in the `java <java>`_ directory (the V2 versions are in the `java2 <java2>`_ directory).
+The code examples are organized by programming language. For example, all of the code examples for
+the latest version of the `AWS SDK for Java 2.x Developer Guide <https://github.com/awsdocs/aws-java-developer-guide-v2>`_
```

Comment:
Would it be better to link to the doc version (https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/home.html) instead of the GitHub version of the SDK guide?